tests/manage/z_cluster/test_drain_node_mon.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import pytest
+
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check
+from ocs_ci.ocs.resources.pod import get_mon_pods
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes, get_node_objs
+from ocs_ci.helpers.helpers import get_mon_pdb
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_external_mode
+@skipif_ocs_version("<4.6")
+@bugzilla("1959983")
+class TestDrainNodeMon(ManageTest):
```

Comment:
Please run Background IOs and check that there are no unexpected IO failures after draining.

Reply:
We run IO during productions runs, so that should be enough is what I feel

Reply:
we run IO during tier1, OCP upgrade and OCS upgrade runs only at this point